duckduckgo/duckduckgo-privacy-extension
# Remove message getShareFeedbackLink while message handler is not ready

**Reviewer:**



## Description:
We were requesting the Feedback Form url through a message `getShareFeedbackLink`, but since this message handler is not ready yet, it was throwing errors in the background.


## Steps to test this PR:
Build steps: https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/CONTRIBUTING.md#building-the-extension

1. Access https://privacy-test-pages.glitch.me/privacy-protections/youtube-click-to-load/2. 
2. Inspect background page
3. It should not throw an error `Unrecognized message to background: {messageType: 'getShareFeedbackLink', options: ''}`

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
